dev-python/spyder/spyder-2.3.8.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+#
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python IDE with matlab-like features"
+HOMEPAGE="https://github.com/spyder-ide/spyder"
+SRC_URI="https://bitbucket.org/spyder-ide/spyderlib/downloads/spyder-2.3.8.zip"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc ipython matplotlib numpy pep8 +pyflakes pylint qtconsole +rope scipy sphinx"
+
+# rope requires no version bordering since all are >= miniumum version
+RDEPEND="
+	|| ( dev-python/PyQt4[${PYTHON_USEDEP},svg,webkit]
+		 dev-python/pyside[${PYTHON_USEDEP},svg,webkit] )
```

Comment:
I kept them the way they were in the previous ebuild (spyder-2.3.5.2). Should I remove lines 6, 8, 10, 14 and 19?